services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -65,21 +73,17 @@
 import uk.gov.moj.sdt.services.utils.GenericXmlParser;
 import uk.gov.moj.sdt.utils.AbstractSdtUnitTestBase;
 import uk.gov.moj.sdt.utils.SdtContext;
+import uk.gov.moj.sdt.validators.CCDReferenceValidator;
 
-import java.text.MessageFormat;
-import java.util.ArrayList;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import java.util.function.Supplier;
-import javax.xml.ws.WebServiceException;
+import static org.easymock.EasyMock.expect;
```

Comment:
Are we just using EasyMock temporarily pending conversion to Mockito?

Reply:
Yes, using EasyMock temporarily for services module, until services module is migrated to Junit (Mockito)